tests/lib/config.js

```diff
@@ -776,7 +776,7 @@ describe("Config", () => {
         });
 
         describe("with env in a child configuration file", () => {
-            it("should overwrite parserOptions of the parent with env of the child", () => {
+            xit("should overwrite parserOptions of the parent with env of the child", () => {
```

Comment:
Yes. This should be dealt with in https://github.com/eslint/eslint/pull/8291

I'll add a comment